### DISCUSSION TITLE
怎么修改搜索框的样式？

### DISCUSSION THREAD

**Discussion by EarOfWheat** [gdh1995/vimium-c#1153] [Q&A]
我想把UI变成直角，但发现设置里的“自定义界面样式”似乎不能改搜索框的边缘(border-radius)。我尝试在电脑文件夹里修改插件的front/vomnibar.html，但插件会崩溃关闭（或者edge会检查篡改？）。也许设置的“定制搜索框页面”可以改，但**我是小白**不会。

**Comment by gdh1995** [comment]
Wiki 里有篇英文说明 https://github.com/gdh1995/vimium-c/wiki/Style-the-UI-of-Vimium-C-using-custom-CSS#basic-structure

浏览器确实会检查扩展文件夹里文件的完整性，所以一般情况不建议这么改。特殊需求可以参考 https://github.com/gdh1995/vimium-c/wiki/Inject-into-other-extensions 来用开发模式加载修改后的扩展文件夹。
